tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
+
+This tutorial will explain how to get notified by [Slack](https://slack.com/) of various events that originate within [Google Cloud Platform](https://cloud.google.com/). `Google Compute Engine` instances may start and stop, new `Google AppEngine` verions might be deployed and `Google Cloud Storage` bucket may be created or deleted. People would like to be notified of these and others via their `Slack` account.
+
+`gSlack` allows you to install a lightweight serverless solution into your
+own Google Cloud Platform project/s and easily configure the alerts you would like to receive.
+
+## Objectives
+In this tutorial you will learn how to:
+- Enable `Slack` integration.
```

Comment:
Slack and other proper nouns in this section don't need to be fixed-width font. (Remove quotes)